package.json

```
@@ -30,10 +30,7 @@
     "storybook": "start-storybook -p 9001 -c .storybook",
     "storybook:build": "build-storybook -c .storybook -o website/stories",
     "examples:build": "yarn build && ./examples/build.sh",
-    "docs:build": "cd docs && yarn && yarn build",
-    "docs:build:dev": "cd docs && yarn && yarn build:dev",
-    "docs:watch": "cd docs && yarn && yarn dev",
-    "docs:deploy": "cd docs && yarn && yarn deploy",
+    "docs:build": "yarn storybook:build && yarn examples:build",
```

We use this command in IS.js:

https://github.com/algolia/instantsearch.js/blob/bb5b9ee1f348bb45ad778b524bce4bc2ec6dd622/package.json#L32

```suggestion
    "website:build": "yarn storybook:build && yarn examples:build",
```